src/frontend/typer/toggles.ts

```diff
@@ -16,6 +16,7 @@ export enum ToggleNavn {
     støtterEnsligMindreårig = 'familie-ba-sak.behandling.enslig-mindreaarig',
     manuellPostering = 'familie-ba-sak.manuell-postering',
     støtterRefusjonEøs = 'familie-ba-sak.behandling.refusjon-eos',
+    organiserAvslag = 'familie-ba-sak-frontend.vedtaksperiode-organisering',
```

Comment:
Er det sånn i BA at denne togglen må legges inn i `familie-ba-sak` sin toggle-controller for at den skal komme til frontend?